ui-components/src/notifications/ApplicationStatus.tsx

```diff
@@ -5,30 +5,40 @@ import "./ApplicationStatus.scss"
 
 export interface ApplicationStatusProps {
   content: string
-  subContent?: string
+  iconColor?: string
+  iconType?: IconTypes
   status?: ApplicationStatusType
+  subContent?: string
   vivid?: boolean
   withIcon?: boolean
-  iconType?: IconTypes
 }
 
 const ApplicationStatus = (props: ApplicationStatusProps) => {
   let bgColor = ""
+  const {
+    content,
+    iconColor,
+    iconType = "clock",
+    status = ApplicationStatusType.Open,
+    subContent,
+    vivid,
+    withIcon = true,
+  } = props
+
   // determine styling
-  const vivid = props.vivid || false
   let textColor = vivid ? "text-white" : "text-gray-800"
   const textSize = vivid ? "text-xs" : "text-sm"
 
-  const status = props.status || ApplicationStatusType.Open
-  const content = props.content
-  const withIcon = props.withIcon ?? true
-  const iconType = props.iconType ?? "clock"
-
-  let icon
-
-  if (withIcon) {
-    icon = <Icon size="medium" symbol={iconType} fill={vivid ? IconFillColors.white : undefined} />
-  }
+  const icon = withIcon && (
+    <span>
+      <Icon
+        size="medium"
+        symbol={iconType}
+        fill={iconColor || (vivid ? IconFillColors.white : undefined)}
+      />{" "}
+      &nbsp;
+    </span>
```

Comment:
I'm not sure we want to take on the added overhead of adding the required space here. it feels like breaking change territory and if it's not something we need for this story maybe we should hold off so we can move this one ahead?

Reply:
Thanks for pointing this out! I did not intend to add that in and even wanted to remove the br tag below since neither are semantically correct. Something weird happened in my ide and I thought I had fixed it. I will revert to the previous markup.